src/Fantomas/Utils.fs

```diff
@@ -2,6 +2,13 @@
 
 open System
 open System.Text.RegularExpressions
+open FSharp.Compiler.Ast
+
+[<RequireQualifiedAccess>]
+module SynExpr =
```

Comment:
Does it not make more sense to move this to SourceParser or SourceTransformer?
@jindraivanek this feels inconsistent.

Reply:
Yes, SourceTransformer looks like good place for this.

Reply:
Deleted entirely, found SourceTransformer.hasParenthesis that does exactly what I need.